apisupport/maven.apisupport/src/org/netbeans/modules/maven/apisupport/NBMNativeMWI.java

```diff
@@ -103,14 +102,11 @@ public void run() {
             if (packageName != null) {
                 String path = packageName.replace(".", "/") + "/Bundle.properties";
                 mf.setAttribute("OpenIDE-Module-Localizing-Bundle", path, null);
-                BufferedOutputStream bos = null;
-                try {
-                    bos = new BufferedOutputStream(new FileOutputStream(new File(src, "manifest.mf")));
+                try (FileOutputStream fos = new FileOutputStream(new File(src, "manifest.mf"));
+                        BufferedOutputStream bos = new BufferedOutputStream(fos)) {
```

Comment:
wouldn't it be easier to read to use one wrapped stream instead of two?
```java
 try (OutputStream os = new BufferedOutputStream(new FileOutputStream(new File(src, "manifest.mf"))) {
```
or
```java
 try (OutputStream os = new BufferedOutputStream(Files.newOutputStream(Path.of(src, "manifest.mf"))) {
```